scss/forms/_form-check.scss

```diff
@@ -262,14 +262,6 @@
       height: 100%;
       content: "";
     }
-
-    &:not(:first-of-type) {
-      border-left: 0;
-    }
-
-    &:not(:last-of-type) {
```

Comment:
Maybe we can keep those blocks but update the selector to `.btn-group &:not(...)` ?
Seems to work on my side and would more understandable to me.